pygmt/src/info.py

```diff
@@ -7,7 +7,7 @@
 
 
 @fmt_docstring
-@use_alias(C="per_column", I="spacing", T="nearest_multiple", V="verbose", f="coltypes")
+@use_alias(C="per_column", I="sequence", T="nearest_multiple", V="verbose", f="coltypes")
```

Comment:
The change is incorrect. The `@use_alias` define the `spacing` parameter as an alias of **I**. And the `kwargs_to_strings` defined that **I** (i.e., `spacing`) can be a *sequence*, i.e, `spacing=(3, 2)` will be converted to `-I3/2`.
```suggestion
@use_alias(C="per_column", I="spacing", T="nearest_multiple", V="verbose", f="coltypes")
@kwargs_to_strings(I="sequence")
```

Reply:
I'm sorry, overlooked the decorator... Thanks for the fix @seisman.